openjdk/excludes/ProblemList_openjdk11.txt

```diff
@@ -40,7 +40,7 @@ compiler/gcbarriers/UnsafeIntrinsicsTest.java#shenandoah https://github.com/adop
 java/lang/Class/GetPackageBootLoaderChildLayer.java	https://github.com/adoptium/aqa-tests/issues/1267	macosx-all
 java/lang/ProcessBuilder/checkHandles/CheckHandles.java https://github.com/adoptium/aqa-tests/issues/1773 windows-x86
 # java/lang/String/StringRepeat.java https://bugs.openjdk.java.net/browse/JDK-8221400
-java/lang/String/StringRepeat.java	https://github.com/adoptium/aqa-tests/issues/1272 windows-all
+java/lang/String/StringRepeat.java#id1	https://github.com/adoptium/aqa-tests/issues/1272 windows-all
```

Comment:
OK I'll bite, hows does #id1 work?

Reply:
The upstream jtreg tool expects this format, when it reads in and parses these problemlist files.  (some testcases need this specificity in order for the testcase to be successfully parsed and excluded.  I am trying to find 'good documentation' to point you at, best I can come up with is: https://openjdk.java.net/jtreg/faq.html#multiple-tests and search for "fragment indentifier".